.pre-commit-hooks.yaml

```diff
@@ -1,5 +1,5 @@
 - id: checkov-helm
   name: Checkov - Helm check
-  entry: bash -c "helm template helm/alfresco-content-services --set \"alfresco-search.alfresco-insight-zeppelin.insightzeppelin.enabled\"=true > helm/alfresco-content-services.yaml && checkov -f helm/alfresco-content-services.yaml --framework=kubernetes --skip-check $(cat .checkov-ignore-list) --quiet"
+  entry: checkov -d helm/alfresco-content-services --framework helm --var-file .checkov/helm_vars.yaml --skip-check $(cat .checkov/skip-list) --quiet
```

Comment:
I think you can get rid of this file since it's not really used anywhere else https://github.com/search?l=YAML&q=org%3AAlfresco+checkov-helm&type=Code